code/modules/organs/external/_external_icons.dm

```diff
@@ -1,10 +1,7 @@
 var/global/list/limb_icon_cache = list()
 
-/obj/item/organ/external/set_dir(var/direction, var/forced)
-	SHOULD_CALL_PARENT(FALSE)
-	if(forced)
-		return ..(direction)
-	return FALSE
+/obj/item/organ/external/set_dir()
+	return ..(SOUTH)
```

Comment:
I mean, things do call this with second arg, not sure what's going on here.

Reply:
The second arg doesn't exist in any other set_dir override.

Reply:
Pushed an amended commit, nothing uses the forced arg now.